NEWS_abbr.md

```diff
@@ -0,0 +1,288 @@
+  #> shape: (1, 2)
+  #> ┌─────┬─────┐
+  #> │ x   ┆ y   │
+  #> │ --- ┆ --- │
+  #> │ f64 ┆ str │
+  #> ╞═════╪═════╡
+  #> │ 1.0 ┆ a   │
+  #> └─────┴─────┘
+  ```
+
+  ```r
+  ### NEW
+  pl$DataFrame(data.frame(x = 1, y = "a"))
+  #> shape: (1, 1)
+  #> ┌───────────┐
+  #> │           │
+  #> │ ---       │
+  #> │ struct[2] │
+  #> ╞═══════════╡
+  #> │ {1.0,"a"} │
+  #> └───────────┘
+
+  pl$DataFrame(!!!data.frame(x = 1, y = "a"))
+  #> shape: (1, 2)
+  #> ┌─────┬─────┐
+  #> │ x   ┆ y   │
+  #> │ --- ┆ --- │
+  #> │ f64 ┆ str │
+  #> ╞═════╪═════╡
+  #> │ 1.0 ┆ a   │
+  #> └─────┴─────┘
+  ```
+
+  Use `as_polars_df()` and `as_polars_lf()` to convert existing R `data.frame`
+  to their `polars` equivalents.
+
+* The class names `PTime` and `rpolars_raw_list` (used to handle time and binary
+  variables) are removed. One should use the classes provided in packages
+  `hms` and `blob` instead.
+
+  ```r
+  ### OLD
+  r_df <- tibble::tibble(
+    time = hms::as_hms(c("12:00:00", NA, "14:00:00")),
+    binary = blob::as_blob(c(1L, NA, 2L)),
+  )
+
+  # R to Polars
+  pl_df <- as_polars_df(r_df)
+  pl_df
+  #> shape: (3, 2)
+  #> ┌─────────┬──────────────┐
+  #> │ time    ┆ binary       │
+  #> │ ---     ┆ ---          │
+  #> │ f64     ┆ list[binary] │
+  #> ╞═════════╪══════════════╡
+  #> │ 43200.0 ┆ [b"\x01"]    │
+  #> │ null    ┆ []           │
+  #> │ 50400.0 ┆ [b"\x02"]    │
+  #> └─────────┴──────────────┘
+
+  # Polars to R
+  tibble::as_tibble(pl_df)
+  #> # A tibble: 3 × 2
+  #>    time binary
+  #>   <dbl> <list>
+  #> 1 43200 <rplrs_r_ [1]>
+  #> 2    NA <rplrs_r_ [0]>
+  #> 3 50400 <rplrs_r_ [1]>
+  ```
+
+  ```r
+  ### NEW
+  r_df <- tibble::tibble(
+    time = hms::as_hms(c("12:00:00", NA, "14:00:00")),
+    binary = blob::as_blob(c(1L, NA, 2L)),
+  )
+
+  ## R to Polars
+  pl_df <- as_polars_df(r_df)
+  pl_df
+  #> shape: (3, 2)
+  #> ┌──────────┬─────────┐
+  #> │ time     ┆ binary  │
+  #> │ ---      ┆ ---     │
+  #> │ time     ┆ binary  │
+  #> ╞══════════╪═════════╡
+  #> │ 12:00:00 ┆ b"\x01" │
+  #> │ null     ┆ null    │
+  #> │ 14:00:00 ┆ b"\x02" │
+  #> └──────────┴─────────┘
+
+  ## Polars to R
+  tibble::as_tibble(pl_df)
+  #> # A tibble: 3 × 2
+  #>   time      binary
+  #>   <time>    <blob>
+  #> 1 12:00  <raw 1 B>
+  #> 2    NA         NA
+  #> 3 14:00  <raw 1 B>
+  ```
+
+
+### Other changes
+
+* Passing a Series with a single value in `$with_columns()` now works:
+
+  ```r
+  series <- pl$Series("foo", 1)
+  pl$DataFrame(bar = 1:2)$with_columns(series)
+  ```
+
+* `map_batches()` and `map_elements()` still exist but their usage is
+  discouraged. Those functions are not guaranteed to interact correctly with
+  the streaming engine. To apply functions from external packages or custom
+  functions that cannot be translated to polars syntax, we now recommend
+  converting the data to a `data.frame` and use the parallel framework of
+  `purrr` for efficiency. The vignette "Using custom functions" contains more
```

Comment:
This is the current state in the rewrite as I understand it, but maybe I missed something.

Btw, if this paragraph is correct, then I guess we should also keep `map_elements()`? Currently only `map_batches()` is implemented.